external-crates/move/move-compiler/tests/move_2024/parser/positional_field_access_no_annotations.exp

```diff
@@ -0,0 +1,14 @@
+error[E03010]: unbound field
+  ┌─ tests/move_2024/parser/positional_field_access_no_annotations.move:6:9
+  │
+6 │         y.0_u8
```

Comment:
Do we have a test for `y.1_0` instead of `y.10`? 
What about leading zeros? `y.0001`?

Reply:
We do not. But I will add them. They should "just work" since the field parsing for these is the same function that we use to parse numbers normally.

Reply:
Should we support this though? It feels a bit ... strange? Or maybe we just say fuck it and have the autoformatter replace them to something sane?